pydis_site/apps/api/viewsets/bot/infraction.py

```diff
@@ -184,20 +185,18 @@ def get_queryset(self) -> QuerySet:
         filter_expires_after = self.request.query_params.get('expires_after')
         if filter_expires_after:
             try:
-                additional_filters['expires_at__gte'] = datetime.fromisoformat(
-                    filter_expires_after
-                )
+                expires_after_parsed = datetime.fromisoformat(filter_expires_after)
             except ValueError:
                 raise ValidationError({'expires_after': ['failed to convert to datetime']})
+            additional_filters['expires_at__gte'] = timezone.make_aware(expires_after_parsed)
```

Comment:
`make_aware` uses the current timezone if one is not specified. Is this prone to breaking on systems where the default time is not UTC?

Generally this case seems like a gap in our tests; since our tests run on a system whose timezone is UTC, such mistakes are not caught.

Reply:
Hmm, I thought sure "current timezone" refers to the `TIME_ZONE` setting, so
```py
~/workspace/pydis-site
$ rg TIME_ZONE
pydis_site/settings.py
176:TIME_ZONE = 'UTC'
```
but it seems Django differs there:

> The default time zone is the time zone defined by the [TIME_ZONE](https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-TIME_ZONE) setting.
> 
> The current time zone is the time zone that’s used for rendering.

So we don't use anything that would change the _current_ time zone - see https://docs.djangoproject.com/en/4.0/topics/i18n/timezones/#selecting-the-current-time-zone - but I'd still be open to changing these lines to force usage of UTC?

Reply:
It doesn't seem likely we'd ever change it, so I suppose it is fine as-is. I'll leave it to you to make the final call.

Reply:
Mark, thanks for your trust in me. With b78bdd08, I now pass it explicitly.